cmd/vm_firewall.go

```diff
@@ -135,13 +138,18 @@ func setVirtualMachineSecurityGroups(vm *egoscale.VirtualMachine, sgs []egoscale
 		return fmt.Errorf("wrong type expected %q, got %T", "egoscale.VirtualMachine", resp)
 	}
 
+	return nil
+}
+
+// printVirtualMachineSecurityGroups prints a virtual machine instance security groups to standard output.
+func printVirtualMachineSecurityGroups(vm *egoscale.VirtualMachine) {
+	// Refresh the vm object to ensure its properties are up-to-date
+	vm, _ = getVirtualMachineByNameOrID(vm.ID.String())
```

Comment:
:warning:  `errcheck` won't like it, me neither.

Reply:
Fixed.